runtime/src/eval/mod.rs

```diff
@@ -2,13 +2,18 @@
 mod macros;
 mod system;
 
-pub use system::save_return_value;
+pub use system::{save_return_value, save_created_address};
 use crate::{Handler, Runtime, ExitReason, ExternalOpcode, CallScheme};
 
+/// ...
```

Comment:
Why do you need this comment? It's very strange to see it in code.

Reply:
this comment is required for the build